src/Data/List/Relation/Binary/Permutation/Inductive/Properties.agda

```diff
@@ -233,6 +233,11 @@ module _ {a} {A : Set a} where
     { isCommutativeMonoid = ++-isCommutativeMonoid
     }
 
+  -- Another useful lemma
+
+  shift' : ∀ xs ys {zs : List A} → xs ++ ys ++ zs ↭ ys ++ xs ++ zs
+  shift' xs ys = ↭-sym (++-assoc xs ys _) ⟨ trans ⟩ ++⁺ʳ _ (++-comm xs ys) ⟨ trans ⟩ ++-assoc ys xs _
+
```

Comment:
This proof is hard to read. Could you please format it using the
setoid combinators as e.g. [the proof of `∷↭∷ʳ`](https://github.com/agda/agda-stdlib/blob/a7dae0daf5a1750561ce7e06004a25c703d3e50e/src/Data/List/Relation/Binary/Permutation/Inductive/Properties.agda#L255)